sw/device/lib/testing/csrng_testutils.c

```diff
@@ -98,9 +98,13 @@ status_t csrng_testutils_check_internal_state(
   TRY(csrng_testutils_cmd_ready_wait(csrng));
   dif_csrng_internal_state_t got;
   TRY(dif_csrng_get_internal_state(csrng, kCsrngInternalStateIdSw, &got));
+  uint32_t reseed_counter;
+  TRY(dif_csrng_get_reseed_counter(csrng, kCsrngInternalStateIdSw,
+                                   &reseed_counter));
 
   TRY_CHECK(got.instantiated == expected->instantiated);
   TRY_CHECK(got.reseed_counter == expected->reseed_counter);
+  TRY_CHECK(reseed_counter == expected->reseed_counter);
   TRY_CHECK(got.fips_compliance == expected->fips_compliance);
```

Comment:
Is there a reason for having the `TRY` and `TRY-CHECK`calls inter-weaved like that?

Reply:
I just thought to group the checks and the actual CSR read functions together. Thanks for the review.